README.md

```diff
@@ -33,7 +33,19 @@ Welcome to iceoryx2, the efficient, and ultra-low latency inter-process communic
 middleware. This library is designed to provide you with fast and reliable
 zero-copy and lock-free inter-process communication mechanisms.
 
-iceoryx2 is all about providing a seamless experience for inter-process
+So if you want to communicate efficiently between multiple processes or applications
+iceoryx2 is for you. With iceoryx2, you can:
+
+* Send huge amounts of data using a publish/subscribe, request/response, pipeline
+  or blackboard pattern,
```

Comment:
Should we state that request/response, etc. is currently not available? Else someone looks for the feature and leaves disappointed even though pub-sub is great. Stating that's it not yet available but coming would lead to checking out pub-sub and leaving them with excitement on what is yet to come.